QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeThreeTotes.java

```diff
@@ -20,34 +24,38 @@ protected void runAutonomous() throws InterruptedException,
         // Collect first tote + container
         setClampHeight(1.0f);
         collectTote();
-        setClampHeight(0.0f);
-        setClampOpen(true);
-        drive(nudge.get());
-        setClampOpen(false);
-        setClampHeight(1.0f);
+        if (containerToCollect.get() && collectContainers.get()) {
```

Comment:
Maybe it would be better to have one variable for each container instead of an enable and a select?